### PR TITLE
Fix cerbot command

### PR DESCRIPTION
**Pull request by JackWilb**
Adds the --standalone flag to the certbot command and handles the nginx server as required.

### CODE REVIEW

File: ansible/tasks/lets_encrypt.yml [hunk 1]
```diff
@@ -1,17 +1,29 @@
+# Sets up the let's encrypt certs only on first provision (the certbot steps cert step doesn't run if the cert exists)
+
+# Install certbot and pyth
 - name: Install certbot
   apt:
-    name: [certbot, python-certbot-nginx]
+    name: [certbot]
```
Comment (waxlamp) [inline ansible/tasks/lets_encrypt.yml]:
```suggestion
    name: certbot
```

File: ansible/tasks/lets_encrypt.yml [hunk 2]
```diff
@@ -1,17 +1,29 @@
+# Sets up the let's encrypt certs only on first provision (the certbot steps cert step doesn't run if the cert exists)
+
+# Install certbot and pyth
```
Comment (waxlamp) [inline ansible/tasks/lets_encrypt.yml]:
```suggestion
# Install certbot
```

but honestly, this comment can also be deleted (the task is self-explanatory).

**Review by waxlamp — CHANGES_REQUESTED** [review]

File: ansible/tasks/lets_encrypt.yml (hunk 1 above)
Comment (JackWilb) [reply ansible/tasks/lets_encrypt.yml]: fixed in 88037ac

File: ansible/tasks/lets_encrypt.yml (hunk 2 above)
Comment (JackWilb) [reply ansible/tasks/lets_encrypt.yml]: fixed in 88037ac

**Comment by JackWilb** [comment]
@waxlamp Once I can get this to pass I'll merge

**Review by waxlamp — APPROVED** [review]

**Comment by JackWilb** [comment]
This only passed because the cert was already installed. However, I don't feel like bricking the https on the prod server to test this. I think we should just call it good and monitor it for now. What do you think @waxlamp

**Comment by waxlamp** [comment]
> This only passed because the cert was already installed. However, I don't feel like bricking the https on the prod server to test this. I think we should just call it good and monitor it for now. What do you think @waxlamp

I agree. The situation we are watching for mainly occurs the first time a new cert is being set up, is that right?

**Comment by JackWilb** [comment]
Correct. I'll merge it as it is for now then

**Pull request merged**